fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowProcesslistStmt.java

```diff
@@ -12,9 +12,10 @@
 public class ShowProcesslistStmt extends ShowStmt {
     private static final ShowResultSetMetaData META_DATA =
             ShowResultSetMetaData.builder()
+                    .addColumn(new Column("FeHost", ScalarType.createVarchar(16)))
                     .addColumn(new Column("Id", ScalarType.createType(PrimitiveType.BIGINT)))
                     .addColumn(new Column("User", ScalarType.createVarchar(16)))
-                    .addColumn(new Column("Host", ScalarType.createVarchar(16)))
+                    .addColumn(new Column("ClientHost", ScalarType.createVarchar(16)))
```

Comment:
Should use Host to be compatible with MySQL output

Reply:
will Keep current name "Host"